new/cookbook-abandonment.md

```diff
@@ -0,0 +1,50 @@
+---
+RFC: unassigned
+Author: Nell Shamrell-Harrington <nshamrell@chef.com>
+Status: Draft
+Type: Process
+---
+
+# Cookbook Abandonment On Supermarket
+
+The Chef community needs a process around handling when cookbooks on Supermarket are abandoned.  This occurs when a cookbook owner does not respond to requests to adopt the cookbook, and the cookbook has not been updated in some time.
+
```

Comment:
I am going to be blunt and ask:  is this RFC also intended to cover the case where the cookbook owner has died?  I was tangentially involved in such a situation, and while everyone concerned was very helpful, I would have liked to have had more clarity around the overall process.